BrainPortal/lib/session_helpers.rb

```diff
@@ -52,6 +52,15 @@ def current_project
     @current_project
   end
 
+  # return currently active project id if user can assign to it or own group id. Serves as default destination project for many operation
+  def default_target_group_id
+    if current_user.assignable_group.include? current_project
```

Comment:
This crashes. There is no method `assignable_group` . This means you have not even tried your code at all.

Reply:
I was sure I fixed that typo, sorry